src/Storage/TSDatabaseView.m

```diff
@@ -20,12 +20,10 @@
 
 NSString *TSThreadDatabaseViewExtensionName  = @"TSThreadDatabaseViewExtensionName";
 NSString *TSMessageDatabaseViewExtensionName = @"TSMessageDatabaseViewExtensionName";
-NSString *TSThreadIncomingMessageDatabaseViewExtensionName = @"TSThreadOutgoingMessageDatabaseViewExtensionName";
```

Comment:
Note that this constant was wrong - it was a collision with the next database view.  Removing this still improves startup times, but I don't understand how we didn't notice incorrect behavior.